iOS/Flows/Home/Members/MembersCollectionViewLayout.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  MemberCollectionViewLayout.swift
+//  Jibber
+//
+//  Created by Martin Young on 8/9/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+class MemberCellLayoutAttributes: UICollectionViewLayoutAttributes {
```

Comment:
Very simple custom cell attributes. For now we just want to know if it's centered.